### PR TITLE
doc(blog): adds why workflow blog

### PR DESCRIPTION
**Pull request by coltmcnealy-lh**

### CODE REVIEW

File: docs/blog/2024-09-02-microservices-and-workflow.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+## Why Workflow?
+
+Microservice applications are distributed workflows. This presents operational challenges because there is no "leader" providing oversight in microservice processes. Thankfully, a developer-focused and horizontally-scalable workflow engine like LittleHorse can fill the "leader" role, thus providing oversight and reliability, and taming the complexity of your business processes.
+
+Additionally, using a workflow engine allows you to develop a set of _resuable_ and _modular_ tasks which can be easily dropped into any business workflow with a common API. Rather than accumulating tech debt, workflow engines allow you to accumulate a set of useful lego bricks.
+
+In most existing orginizations there's a long list of API calls required to simply _run_ a workflow.  Training engineers to use all of the new APIs while securely distributing access and permissions causes confusion and slow development cycles.  Workflow engines provide a single API and single system that allows anyone to securely manage, run, and operate complex workflows.
```
Comment (jhosep-marin) [inline docs/blog/2024-09-02-microservices-and-workflow.md:88]: There's a little misspelling here. It should be _organization_ :D

**Comment by jhosep-marin** [comment]
Aside from the small spelling mistake, this blog looks good to me :smile: :heavy_check_mark:

**Review by jhosep-marin — APPROVED** [review]

File: docs/blog/2024-09-02-microservices-and-workflow.md (hunk 1 above)
Comment (yesidevelop) [reply docs/blog/2024-09-02-microservices-and-workflow.md:88]: resuable is also spelled wrong

Comment (coltmcnealy-lh) [reply docs/blog/2024-09-02-microservices-and-workflow.md:88]: Great catch, both of you!

**Pull request merged**